frontend/src/pages/premium.page.tsx

```diff
@@ -216,12 +216,12 @@ const PremiumPromo: NextPage = () => {
       )}
     </LinkButton>
   ) : (
-    <Button
-      disabled={true}
+    <LinkButton
```

Comment:
There's a linting error that you didn't remove the `Button` import at the top :)

Reply:
Oops, thanks for the heads up.